cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/constants/MonitorConfig.java

```diff
@@ -44,7 +44,7 @@ private MonitorConfig() {
    * <code>metadata.max.age.ms</code>
    */
   public static final String METADATA_MAX_AGE_CONFIG = CommonClientConfigs.METADATA_MAX_AGE_CONFIG;
-  public static final long DEFAULT_METADATA_MAX_AGE = 55000L;
```

Comment:
Nit: For variables in this package, we use the following naming convention:
``` 
<config>_CONFIG
DEAFULT_<config>
<config>_DOC
```

Adding this suffix only to the default value violates this convention.
Can we either (1) apply the same suffix to the `METADATA_MAX_AGE_CONFIG` and `METADATA_MAX_AGE_DOC` or (2) rollback the name change?

Reply:
I changed `METADATA_MAX_AGE_CONFIG` to `METADATA_MAX_AGE_MS_CONFIG` and `METADATA_MAX_AGE_DOC` to `METADATA_MAX_AGE_MS_DOC`